quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/VaultKeystorePropertyMappers.java

```diff
@@ -0,0 +1,32 @@
+package org.keycloak.quarkus.runtime.configuration.mappers;
+
+import org.keycloak.config.VaultOptions;
+
+import static org.keycloak.quarkus.runtime.configuration.mappers.PropertyMapper.fromOption;
+
+final class VaultKeystorePropertyMappers {
```

Comment:
Why a separate mapper type and not just re-use the `VaultPropertyMappers`? 

Same for keeping the file vault ...